auto-update-dataset/python/README.md

```diff
@@ -0,0 +1,16 @@
+0. We can use ``repoArchivedCheck.py`` to find archived projects in [pr-data.csv](https://github.com/TestingResearchIllinois/idoft/blob/main/pr-data.csv).
+
+1. Environment:  
+    - python3.6  
+
+2. Libraries:  
+    - pandas
+    - requests_html
+
+3. Usage:
+    - ``python3 repoArchivedCheck.py``
+
+4. The example results in console:  
+    - if no update on ```status``` is needed, it'll print ```No need to update```, otherwise "May need to update the STATUS" with details of the repo;
```

Comment:
Thanks for these changes!

Can you improve the output so that one can just update the relevant line numbers with the outputted lines? i.e., instead of outputting the contents of the lines in some array format, output the content in the csv format so that one can just copy and paste the line and the warnings from this script should go away. 

E.g., 
```
750: https://github.com/apache/nifi-registry,f1e5aef7bef7d4f5ec89b2347e6a6c7e78aa589c,nifi-registry-core/nifi-registry-web-api,org.apache.nifi.registry.security.authorization.ResourceAuthorizationFilterSpec.secure requests to an unguarded resource are allowed without an authorization check,UD, RepoArchived,,the recorded repo is archived and it has been moved under the Apache NiFi codebase as nifi-registry(https://github.com/apache/nifi/tree/main/nifi-registry)
```

The same comment applies to all lines outputted by the anomaly section as well.

Lastly, for this entry (`org.apache.nifi.registry.security.authorization.ResourceAuthorizationFilterSpec.secure requests to an unguarded resource are allowed without an authorization`) in pr-data.csv, please look to remove the notes and just use the `RepoArchived` status.

Reply:
some updates:
- 1.changed output format in repoArchieved section and anomaly section 
![image](https://user-images.githubusercontent.com/46290389/142752980-61c40636-fc64-4d62-bf2f-662a8e011e7a.png)

- 2.for ```nifi-registry-core/nifi-registry-web-api```, notes removed and ```RepoArchieved``` status used.